drizzlepac/hlautils/analyze.py

```diff
@@ -246,6 +246,7 @@ def analyze_data(input_file_list, log_level=logutil.logging.NOTSET):
         no_proc_value = None
         do_process = True
         # Imaging vs spectroscopic or coronagraphic
+        print("******OBSTYPE: {}".format(obstype))
```

Comment:
Should this be a log message (log.debug? log.info?) or removed altogether?

Reply:
Unnecessary print statement.